### DISCUSSION TITLE
electron_rom / electron_cart

### DISCUSSION THREAD

**Discussion by motoschifo** [mamedev/discussions#79] [General]
The electron_rom and the electron_cart software lists has different file name using the same information and CRC/SHA1. The only difference is the first capital letter.

For example starspell romset in electron_rom has **Starspell-1.00.rom** (note the capital **S**):
```
	<software name="starspell">
		<description>StarSpell</description>
		<year>1989</year>
		<publisher>Slogger</publisher>
		<part name="rom" interface="electron_rom">
			<dataarea name="rom" size="16384">
				<rom name="Starspell-1.00.rom" size="16384" crc="765a71f3" sha1="a02a8ac35aaafef27a1911161c3c06b509ad56a4"/>
			</dataarea>
		</part>
	</software>
```

and the electron_cart has **starspell-1.00.rom**:
```
	<software name="starspell">
		<description>Starspell</description>
		<year>1989</year>
		<publisher>Slogger</publisher>
		<part name="cart" interface="electron_cart">
			<dataarea name="uprom" size="16384">
				<rom name="starspell-1.00.rom" size="16384" crc="765a71f3" sha1="a02a8ac35aaafef27a1911161c3c06b509ad56a4"/>
			</dataarea>
		</part>
	</software>
```

Is this to be considered correct?
On a Windows file system this can generate an issue.
If this is not the right place to ask that kind of questions, please let me know where I should go.

**Comment by rb6502** [comment]
How can this cause an issue on any file system?  electron_rom and electron_cart are separate directories, and you're supposed to have the items zipped / 7zipped anyway.

**Comment by cuavas** [comment]
There’s no reason for labels to be unique between different software items.  That would be an unreasonable restriction.

**Comment by motoschifo** [comment]
If the rom file is the same (size, CRC, SHA1), also the name should be the same.
No problem if this it's the desired behaviour, I reported because it's strange to have a rom with different names and I thought that was a typo.
If it's ok, this conversation can be marked as closed.
Thanks
